### DISCUSSION TITLE
GHOST Clients Saving to File Shares

### DISCUSSION THREAD

**Discussion by zveltri** [cmu-sei/GHOSTS#96] [Q&A] [answered]
Hello,

I am attempting to run GHOSTS on a local network and I need to test out some access to file shares etc. Is there any way to change where GHOSTS saves?

**Comment by sei-dupdyke** [comment, marked as answer]
Hi, yes, you can change that - this line in the timeline.json file indicates where to store docs (on a per handler basis, this one is just for MS Word) - https://github.com/cmu-sei/GHOSTS/blob/01aa57495849d725005a43b9b7ff0155aca6e14e/src/Ghosts.Client/config/timeline.json#L99

if you want to save to another directory or a file share, change the `"%homedrive%%homepath%\\Documents` path to whatever you like.

**Reply by zveltri** [reply]
So for a network server, the path would be:

[ "%homedrive%%homepath%\\ExampleServer\fileshare", "pdf", "pdf-vary-filenames" ],


Thanks for the help.

**Comment by sei-dupdyke** [comment]
Should just be `[ "\\ExampleServer\fileshare", "pdf", "pdf-vary-filenames" ]`

**Comment by sei-dupdyke** [comment]
("%homedrive%%homepath%" are windows system variables for "your current home drive" and whatever your home settings are)
